src/pages/Home/components/FavoritesCitiesWeatherWidget/FavoritesCitiesWeatherWidget.tsx

```diff
@@ -0,0 +1,31 @@
+import { useFavorites } from 'context/favorites/FavoritesProvider';
+import { Icons } from 'enums/Icons';
+import { CitiesWeatherWidget } from 'components/CitiesWeatherWidget';
+import { CityWeatherShortcut } from 'components/CityWeatherShortcut';
+import { FavoritesCitiesWeatherWidgetTestIds } from './FavoritesCitiesWeatherWidgetTestIds';
+
+const FavoritesCitiesWeatherWidget = () => {
+  const { state } = useFavorites();
+
+  const renderContent = () => {
+    if (state.favorites.length > 0)
+      return (
+        <>
+          <CitiesWeatherWidget.Title>favorites</CitiesWeatherWidget.Title>
+          <CitiesWeatherWidget.ShortcutsWrapper>
+            {state.favorites.map((city) => (
+              <CityWeatherShortcut key={city} cityName={city} icon={Icons.Heart} />
```

Comment:
**todo** We should change Heart icon to Cross icon. Also, we should handle that in `FavoritesButton.tsx`

Reply:
This is FEA-91 task